docs/installation/configuration.rst

```diff
@@ -42,6 +42,25 @@ DATABASES
    `Django documentation on databases <https://docs.djangoproject.com/en/2
    .1/ref/settings/#databases>`_
 
+DEFAULT_FILE_STORAGE
+^^^^^^^^^^^^^^^^^^^^
+
+   By default, Pulp uses the local filesystem to store files. This can be configured though to
```

Comment:
This would be clearer if the first paragraph was all about the defaults and the second paragraph was all about S3. That way more backends could be added as their own paragraphs later too. What do you think?